models/relations.py

```diff
@@ -60,10 +69,13 @@ def json(self):
             "birth_date": self.birth_date.strftime("%m/%d/%Y") if self.birth_date else "", 
             "lives_in": self.lives_in, 
             "nickname": self.nickname
-        }
+        }   
 
 
     @classmethod
     def find_by_user_id(cls, user_id):
         return cls.query.filter_by(user_id = user_id).all()
 
+    @classmethod
```

Comment:
unnecessary function. does the same job as find_by_id, which should have been inherited from the parent function, CommonModel